Mathlib/CategoryTheory/Monoidal/Mod_.lean

```diff
@@ -129,3 +128,42 @@ def comap {A B : Mon_ C} (f : A ⟶ B) : Mod_ B ⥤ Mod_ A where
 -- Lots more could be said about `comap`, e.g. how it interacts with
 -- identities, compositions, and equalities of monoid object morphisms.
 end Mod_
+
+section Mod_Class_
+
+open CategoryTheory Mon_Class MonoidalCategory
+
+variable (M : C) [Mon_Class M]
+
+/--
+An action of a monoid object `M` on an object `S` is the data of map
+`smul : M ⊗ S ⟶ S` that satisfies "the right commutative diagrams" -/
+class Mod_Class_ (S : C) where
+  /-- The action map -/
+  smul : M ⊗ S ⟶ S
+  mul_smul' : (𝟙 M ⊗ smul) ≫ smul
+    = (α_ M M S).inv ≫ (μ ⊗ (𝟙 S)) ≫ smul := by aesop_cat
+  one_smul' : (λ_ S).inv ≫ η ▷ S ≫ smul = 𝟙 S := by aesop_cat
+
+namespace Mod_Class_
+
+@[inherit_doc] notation "γ" => Mod_Class_.smul
+
+/- The simp attribute is reserved for the unprimed versions. -/
+attribute [reassoc] mul_smul' one_smul'
+
+@[reassoc (attr := simp)]
+lemma mul_smul (S : C) [Mod_Class_ M S] : (𝟙 M ⊗ γ) ≫ γ
+    = (α_ M M S).inv ≫
+      (μ ⊗ (𝟙 S) : (M ⊗ M) ⊗ S ⟶ M ⊗ S) ≫ γ := mul_smul'
```

Comment:
Please start multiline return types on a new line
```suggestion
lemma mul_smul (S : C) [Mod_Class_ M S] :
    𝟙 M ⊗ γ ≫ γ = (α_ M M S).inv ≫ (μ ⊗ 𝟙 S : (M ⊗ M) ⊗ S ⟶ M ⊗ S) ≫ γ := mul_smul'
```

Reply:
I guess we don't need this anymore right?